experiments/where-do-users-find-our-sites.md

```diff
@@ -0,0 +1,103 @@
+# Where do users find out about Alaveteli sites?
+
+Where is our existing traffic being referred from? Where might there be
+potential users that we're not currently reaching?
+
+- [issue#4](https://github.com/mysociety/alaveteli-experiments/issues/4)
+
+## Answer
+
+**TL;DR:** so far, referrer analysis reveals a dominance of social media sites
+for most of our sites with more of a news/campaign-driven audience for the more
+journalistic sites such as Ask The EU. Inbound link analysis inconclusive due to
+the limited features of free reports.
+
+## Analysis
+
+This has turned out to be a more complex question than we first realised so we're
+going to try to update this as we find out more.
+
+We have two types of data available:
+
+### [Referrer data](where-do-users-find-our-sites/inbound-traffic.md)
+
+Data extracted from Google Analytics showing which links the people who are
+visiting our sites have clicked to reach us.
+
+From studying this data we appear to have two types of site - those which get
```

Comment:
The reason for "Search Engine" - and indeed "Social Media" - being included comes down to Google Analytics being a bit odd. DuckDuckGo is almost certainly search traffic but Yahoo could be portal, I supposed - without a full referrer string it's impossible to say